kn/leden/mongo.py

```diff
@@ -20,12 +23,14 @@ def _id(obj):
     raise ValueError
 
 class SONWrapper(object):
-    def __init__(self, data, collection, parent=None):
+    def __init__(self, data, collection, parent=None, detect_race=False):
         self._data = data
         self._collection = collection
         self._parent = parent
+        self._detect_race = detect_race
     def delete(self):
         assert self._data['_id'] is not None
+        # TODO check version
         self._collection.remove({
             '_id': self._data['_id']})
```

Comment:
Aangezien ik geen plek in de code kon vinden waar `delete` daadwerkelijk wordt gebruikt, kon ik het moeilijk testen. Dus heb ik die overgeslagen.